docs/quickstart/expansion.rst

```diff
@@ -113,6 +113,8 @@ The MetalK8s GUI uses :term:`SaltAPI` to orchestrate the deployment.
 Adding a node from the command-line
 -----------------------------------
 
+.. _Creating a manifest:
```

Comment:
Let's use a standard for references: `<book>-<chapter>-<ref_name>`.
In that case, it would become:

```suggestion
.. _quickstart-expansion-manifest:
```